root/backend/server/api/routes/poll.js

```diff
@@ -78,4 +80,39 @@ router.get('/:pollId/pollInfo', async (req, res) => {
   }
 });
 
+router.post('/:pollUuid/response', async (req, res, next) => {
+  const { name, choices } = req.body;
+  const userUuid = crypto.randomBytes(24).toString('hex');
+
+  try {
+    const responsesData = await Poll.findById(req.params.pollUuid).then(data => {
+      return data.responses;
+    });
+  
+    let newResponseObj = {};
+
+    if (responsesData) {
+      newResponseObj = {
+        ...responsesData,
+      }
+    }
```

Comment:
if you're not adding anything else to this, no need to use the spread operator here

```suggestion
    if (responsesData) {
      newResponseObj = responsesData;
    }
```

Reply:
Gotcha, thanks